.github/workflows/locale-updater.yml

```
@@ -0,0 +1,49 @@
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # 履歴を全て取得して差分比較に使用
+          ref: ${{ github.event.pull_request.head.ref }} # PRのブランチをチェックアウト
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.4)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)